gamesinfo.py

```diff
@@ -32,11 +28,17 @@ def get_game_info(threadpool, options, games, cachedgames, keys, gameName,
             # or allow the user to do so in the html page.
             return
 
+        # Whether the cache is ignored or not, if a game cached has a gift_date,
+        # we keep it
+        if ((gameName in cachedgames) and (cachedgames[gameName].hfr.gift_date)):
```

Comment:
Autant avoir un if (gameName in cachedgames) pour les 2 blocs vu que c'est le même :)